apis/goharbor.io/v1alpha2/core_types.go

```diff
@@ -0,0 +1,310 @@
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// +kubebuilder:object:root=true
+// +kubebuilder:storageversion
+// +k8s:openapi-gen=true
+// +resource:path=core
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:categories="goharbor"
+// +kubebuilder:printcolumn:name="Version",type=string,JSONPath=`.spec.version`,description="The semver Harbor version",priority=5
+// +kubebuilder:printcolumn:name="Replicas",type=string,JSONPath=`.spec.replicas`,description="The number of replicas",priority=0
+// Core is the Schema for the registries API.
+type Core struct {
```

Comment:
Is mapping to `harbor-core` component? It seems to define the original `harbor` CR spec.